### ISSUE TITLE
User alert messages use link with an empty 'href' as a close button - A11y issue

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#4117]
**PR is coming!**  

**Branch**: develop

**Current Behavior**: The use of links with an empty `href` should be avoided at all cost as they represent a problem for users with screen readers. The `X` at the right border of user alert messages, which is used to close them, is constructed with a link instead of the more appropriate `button` element. 

![a-close](https://cloud.githubusercontent.com/assets/1457929/8752880/9a59bbda-2cb7-11e5-9f22-b4bc3296560e.png)

**Comment by radinamatic** [comment]
Fixed in #4142, closing.
